Dockerfile

```diff
@@ -20,4 +23,5 @@ CMD dlv -l :40000 --headless=true --api-version=2 test -test.v ./...
 
 FROM alpine as runtime
 COPY --from=build /bin/app /bin/app
+COPY --from=build /bin/dlv /bin/dlv
```

Comment:
Note: you could use target debug.
```suggestion
```

Reply:
Yes, but there is an additional option to start debugging ourselves with passed env variable:
https://github.com/networkservicemesh/cmd-nse-vfio/blob/a01453eceee67e38b8e3434dbf0b73044d7a3f37/main.go#L77-L79

Reply:
I'm not sure that we need dlv in runtiume.